mm/alignment.py

```diff
@@ -121,6 +135,38 @@ def run(self, task_id):
                 self.out_single_word_boundary_caches[task_id].get_path(),
             )
 
+    def plot(self):
+        import numpy as np
+        import matplotlib
+        import matplotlib.pyplot as plt
+
+        # Parse the files and search for the average alignment score values (normalized over time).
+        alignment_scores = []
+        for log_file in self.out_log_file.values():
+            logging.info("Reading: {}".format(log_file))
+            file_path = tk.uncached_path(log_file)
+            document = ET.parse(util.uopen(file_path))
+            _seg_list = document.findall(".//segment")
+            for seg in _seg_list:
+                avg = seg.find(".//score/avg")
+                alignment_scores.append(float(avg.text))
+            del document
+
+        np_alignment_scores = np.asarray(alignment_scores)
+        logging.info(
+            f"Max {np_alignment_scores.max()}; Min {np_alignment_scores.min()}; Median {np.median(np_alignment_scores)}"
```

Comment:
maybe also add the total number of segments/alignment scores and the number of segments above the 90 percentile, since those are not taken for plotting?

Reply:
Sure, I can do that.

I wonder if it would be interesting to add an output file consisting of all segments along with their estimated percentile.